logic/src/commonMain/kotlin/com/wire/kalium/logic/data/connection/ConnectionMapper.kt

```diff
@@ -41,6 +43,16 @@ internal class ConnectionMapperImpl(
         fromUser = otherUser?.let { publicUserMapper.fromDaoModelToPublicUser(it) }
     )
 
+    override fun fromDaoToConnectionDetails(connection: ConnectionEntity, otherUser: UserEntity?): ConversationDetails {
+        return ConversationDetails.Connection(
+            conversationId = idMapper.fromDaoModel(connection.qualifiedConversationId),
+            otherUser = otherUser?.let { publicUserMapper.fromDaoModelToPublicUser(it) },
+            userType = UserType.EXTERNAL, // todo(conn): map it
```

Comment:
will this be addressed in this PR or another one?

Reply:
Both of you were right, I merged the changes from @trOnk12 and added the mapping logic ! 🙌